utils/scripts/ocm/ocm.py

```diff
@@ -860,6 +929,29 @@ def update_notification_email_address(self, addon_name, email_address, exit_on_f
 
         subparsers = parser.add_subparsers(title='Available sub commands',
                                            help='sub-command help')
+        
+        # Argument parsers for get ods_latest version
+        get_latest_osd_candidate_json = subparsers.add_parser(
+            'get_latest_osd_candidate_json',
+            help="Parser to get osd chnages in json file",
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        get_latest_osd_candidate_json.add_argument("--json_path",
+                                                help="json file path, The file should be created already",
```

Comment:
json file path to store osd latest version details. The file should be created already.